src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_MAX = 100;
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figures [] figures = Figures.values();
+        int index = random.nextInt(figures.length);
+        switch (figures[index]) {
+            case CIRCLE:
+                return getRandomCircle();
+            case SQUARE:
+                return getRandomSquare();
+            case RECTANGLE:
+                return getRandomRectangle();
+            case RIGHT_TRIANGLE:
+                return getRandomRightTriangle();
+            case ISOSCELES_TRAPEZOID:
+                return getRandomIsoscelesTrapezoid();
+            default:
+        }
+        return null;
```

Comment:
```suggestion
        switch (figures[index]) {
            case CIRCLE:
                return getRandomCircle();
            case SQUARE:
                return getRandomSquare();
            case RECTANGLE:
                return getRandomRectangle();
            case RIGHT_TRIANGLE:
                return getRandomRightTriangle();
            case ISOSCELES_TRAPEZOID:
            default:
                return getRandomIsoscelesTrapezoid();
        }
```
returning null is a bad practice here - because this method should return a random figure